src/org/usfirst/frc/team2706/robot/RobotMap.java

```diff
@@ -162,6 +162,13 @@
     
     private static final String ROBOT_ID_LOC = "/home/lvuser/robot-type.conf";
 
+    // Bling Various
+    // Number of pixels in LED Strip
+    public static final int LED_Strip_PixelCount = 120;
```

Comment:
This isn't really how RobotMap is meant to be used.
The RobotMap constants end before `ROBOT_ID_LOC`
```Java
    // Bling Various
    // Number of pixels in LED Strip
    private static final int[] LED_STRIP_PIXELCOUNT_VALS = {120, 120, 120};
    public static final int LED_STRIP_PIXELCOUNT = getConstant("LED_STRIP_PIXELCOUNT ");
   
    // Battery Variable which equals the voltage of the battery
    private static final double[] BATTERY_CAPACITY_VALS = {12.0, 12.0, 12.0};
    public static final double BATTERY_CAPACITY = getConstant("BATTERY_CAPACITY ");
```